src/utils/logging.js

```diff
@@ -0,0 +1,18 @@
+// A-Frame blows away any npm debug log filters so this allow the user to set the log filter
+// via the query string.
+import debug from "debug";
+
+const qs = new URLSearchParams(location.search);
+
+function qsTruthy(param) {
+  const val = qs.get(param);
+  // if the param exists but is not set (e.g. "?foo&bar"), its value is the empty string.
+  return val === "" || /1|on|true/i.test(val);
+}
```

Comment:
`qsTruthy` is already defined in `hub.js`

Reply:
i'll just break it out. i've seen it copy/pasted elsewhere as well